test/integration/puppetlabs/services/master/master_service_test.clj

```diff
@@ -777,38 +777,56 @@
                     :master-conf-dir master-service-test-runtime-dir}}
 
     (testing "can retrieve file_content from the modules mount from a project"
-      (let [response (http-get "/puppet/v3/file_content/modules/utilities/etc/greeting?project=local")]
+      (let [response (http-get "/puppet/v3/file_content/modules/utilities/etc/greeting?project=local-23")]
```

Comment:
I think we're using an underscore separator now. Doesn't really matter since you're using fixtures, but it might be good to have it reflect how things will really look, so as not to confuse ourselves :)

Reply:
:disappointed:

Reply:
Thanks. Sorry about that. Still think it was a little unnecessary that were asked to change that in the first place, but oh well.